src/DssSpell.sol

```diff
@@ -60,222 +58,171 @@ contract DssSpellAction is DssAction {
-        DssExecLib.sendPaymentFromSurplusBuffer(IMMUNEFI_COMISSION, 10_000);
+        // RWA009_A_INPUT_CONDUIT_URN_USDC
+        DssExecLib.setContract(RWA009_A_INPUT_CONDUIT_URN_USDC , "psm", MCD_LITE_PSM_USDC_A);
 
-        // ---------- Update ClipperMomJob ----------
-        // Forum: https://forum.makerdao.com/t/executive-inclusion-clippermomjob-update/24774
+        // ----- Phase 2 USDC Migration from PSM-USDC-A to LITE-PSM-USDC-A -----
+        // Forum: https://forum.makerdao.com/t/lite-psm-usdc-a-phase-2-major-migration-proposed-parameters/24839
+        // Poll: https://vote.makerdao.com/polling/QmU7XJ6X
 
-        // remove the old ClipperMomJob from the CronSequencer
-        DssCronSequencerLike(CRON_SEQUENCER).removeJob(CRON_CLIPPER_MOM_JOB);
+        // ----- Update PSM-USDC-A Fees -----
+        // Forum: https://forum.makerdao.com/t/lite-psm-usdc-a-phase-2-major-migration-proposed-parameters/24839
+        // Poll: https://vote.makerdao.com/polling/QmU7XJ6X
 
-        // add the new ClipperMomJob (0x7E93C4f61C8E8874e7366cDbfeFF934Ed089f9fF) to the Cron Sequencer
-        DssCronSequencerLike(CRON_SEQUENCER).addJob(CRON_CLIPPER_MOM_JOB_NEW);
+        // ----- Update PSM-USDC-A DC-IAM -----
+        // Forum: https://forum.makerdao.com/t/lite-psm-usdc-a-phase-2-major-migration-proposed-parameters/24839
+        // Poll: https://vote.makerdao.com/polling/QmU7XJ6X
 
-        // Update CRON_CLIPPER_MOM_JOB to 0x7E93C4f61C8E8874e7366cDbfeFF934Ed089f9fF in the Chainlog
-        DssExecLib.setChangelogAddress("CRON_CLIPPER_MOM_JOB", CRON_CLIPPER_MOM_JOB_NEW);
+        // ----- Update MCD_LITE_PSM_USDC_A Buf -----
+        // Forum: https://forum.makerdao.com/t/lite-psm-usdc-a-phase-2-major-migration-proposed-parameters/24839
+        // Poll: https://vote.makerdao.com/polling/QmU7XJ6X
 
-        // Note: bump chainlog version due to the updated CRON_CLIPPER_MOM_JOB address
-        DssExecLib.setChangelogVersion("1.17.6");
+        // ----- Update LITE-PSM-USDC-A DC-IAM -----
+        // Forum: https://forum.makerdao.com/t/lite-psm-usdc-a-phase-2-major-migration-proposed-parameters/24839
+        // Poll: https://vote.makerdao.com/polling/QmU7XJ6X
 
-        // ---------- Aligned Delegate MKR Compensation ----------
-        // Forum: https://forum.makerdao.com/t/july-2024-aligned-delegate-payment-requests/24794
-        // MIP: https://mips.makerdao.com/mips/details/MIP101#2-6-3-aligned-delegate-budget-and-participation-requirements
+        // Note: load the MCD contracts depencencies
+        DssInstance memory dss = MCD.loadFromChainlog(DssExecLib.LOG);
 
-        // BLUE - 13.75 MKR - 0xb6C09680D822F162449cdFB8248a7D3FC26Ec9Bf
-        MKR.transfer(BLUE, 13.75 ether); // Note: 'ether' is a keyword helper, only MKR is transferred here
+        // Note: specify the migration config
+        DssLitePsmMigrationConfigPhase2 memory cfg = DssLitePsmMigrationConfigPhase2({
+            // Note: chainlog key of new psm lite
```

Comment:
Nit: the existing comment might lead the reader to think that a new contract is being deployed.
```suggestion
            // Note: chainlog key for LITE-PSM-USDC-A
```